modules/forum/src/main/ForumPostApi.scala

```diff
@@ -66,10 +66,10 @@ final class ForumPostApi(
               timeline ! Propagate(TimelinePost(me, topic.id, topic.name, post.id)).pipe {
                 _ toFollowersOf me toUsers topicUserIds exceptUser me withTeam categ.team
               }
-            else if categ.id == ForumCateg.diagnosticId then
+            else if categ.isDiagnostic then
               timeline ! Propagate(TimelinePost(me, topic.id, topic.name, post.id)).toUsers(topicUserIds)
             lila.mon.forum.post.create.increment()
-            mentionNotifier.notifyMentionedUsers(post, topic)
+            if !categ.isDiagnostic then mentionNotifier.notifyMentionedUsers(post, topic)
```

Comment:
This is a problem for me. There are a bunch of dead diagnostics threads because I cannot get the attention of the users who posted them, probably because they're on mobile and never see the timeline. If you won't allow pings can you suggest an alternative?